settings/global.mdx

```diff
@@ -404,6 +404,59 @@ Example:
   </Expandable>
 </ResponseField>
 
+### Feedbacks
+
+<ResponseField name="feedback" type="Feedback">
+  Configurations to add a feedback section to the end of every page
+
+   <Expandable title="Redirect">
+    <ResponseField name="suggestEdit" type="boolean">
+    Enable a "Suggest Edit" button that creates an edit to the source documentation in your documentation Github repo, provided that the Github repo is public
+
+    </ResponseField>
+    <ResponseField name="raiseIssue" type="boolean">
+    Enable a "Raise Issue" button that creates an issue in your documentation Github repo, provided that the Github repo is public
+
+    </ResponseField>
+
+    </ResponseField>
+    <ResponseField name="review" type="boolean">
+    Enable a review section where readers can rate if your document is useful or not
+
+    </ResponseField>
+
+  </Expandable>
+
+</ResponseField>
+
+### Repo
+<ResponseField name="repo" type="Repo">
+  Specify the repo where you store your source documentations
+
+  <Expandable title="Repo">
+     <ResponseField name="github" type="github_repo">
+        Information about your documentation Github Repo
+        <Expandable title="Redirect">
+          <ResponseField name="owner" type="string">
+          </ResponseField>
+          <ResponseField name="repo" type="string">
+          </ResponseField>
+          <ResponseField name="deployBranch" type="string">
+            The deployment branch name of your documentation
+          </ResponseField>
+          <ResponseField name="contentDirectory" type="string">
+            The name of the root directory where your documentation is stored in the Github repo
+          </ResponseField>
+          <ResponseField name="isPrivate" type="boolean">
+            Whether your documentation repo is private. 
+
+            Note: if you enabled Suggest Edits or Raise Issue in the feedback section, they would only work if your documentation Github repo is public.
+          </ResponseField>
+        </Expandable>
+     </ResponseField>
+  </Expandable>
+</ResponseField>
+
```

Comment:
This is a little confusing, but repo isn't actually configured in the mint.json. It's configured somewhere else, not exactly sure where though